src/test/suite/fixtures/recorded/languages/rust/changeValueCap.yml

```diff
@@ -0,0 +1,32 @@
+languageId: rust
```

Comment:
Oh ha man how did we miss this one 😂.  Good catch

Worth also adding a "chuck" test for this one to make sure it removes the `=`, as that will just leave you with a variable declaration, right?

Reply:
Not really. You can’t declare a `const ` without providing some value for it.

Reply:
Ah right.  So we just support "chuck value" removing `=` for non-const then?  makes sense to me